libs/sdk-ui-kit/src/Button/typings.ts

```diff
@@ -22,5 +36,10 @@ export interface IButtonProps {
     variant?: "primary" | "secondary";
     intent?: "action" | "positive" | "negative";
     size?: "small" | "medium" | "large";
-    accessibilityConfig?: IAccessibilityConfigBase;
+    /**
+     * @deprecated use `accessibilityConfig.ariaLabel` instead
+     */
+    ariaLabel?: string;
```

Comment:
Will this be removed in the next pull request or do we need to live with it?

Reply:
removed